src/User_Command.php

```diff
@@ -545,6 +545,10 @@ public function update( $args, $assoc_args ) {
 			$user_ids[] = $user->ID;
 		}
 
+		if ( empty( $user_ids ) ) {
+			exit( 1 );
```

Comment:
```suggestion
			WP_CLI::error( 'No valid users found.' )
```
If we are exiting unsuccessfully I think we should give a reason and let the user know it is intentional. Otherwise it seems like we either die on warnings, or that something has gone wrong since there is no other output. We can't error on the invalid user itself because there might be more than one, but we can here.

Reply:
@mrsdizzie - Thank you reviewing this PR. The requested changes has been completed. Please have a look.